app/assets/stylesheets/document-component.scss

```diff
@@ -3,7 +3,7 @@
     margin-bottom: 1rem;
   }
   .accordion-button {
-    font-size: 1.7rem;
+    font-size: 2.0rem;
```

Comment:
Bumping a couple font sizes now that Argo is opinionated about fonts. Prior to this PR, Argo appeared to be fine with any of a grab-bag of fonts from Blacklight's defaults, which allow browsers to vary the display wildly.